### ISSUE TITLE
windows 10: window position shifts upon alt-click, BEFORE dragging takes place

### ISSUE THREAD

**Issue by reikred** [stefansundin/altdrag#43]
I have a new win10 laptop and installed altdrag1.1. When I do alt-click  to prepare for a drag operation, the window instantly  moves (jumps) maybe 2 inches toward upper left screen corner, so that quite a bit of the window is off-screen. This occurs WITHOUT moving the mouse at all. I can drag the window back to were I want it, but what I'm trying to say is that the position shifted by quite a bit BEFORE I started any dragging.

I also updated my win7 desktop to altdrag1.1, and the described behavior does NOT occur there.

(In X11 terminology, I would perhaps describe the behavior as the window being "warped" toward the UL corner of the display, just to say it in a different way.)

**Comment by reikred** [comment]
May be related to https://github.com/stefansundin/altdrag/issues/7#issuecomment-50701359

I tried the suggested workaround. It now works as it should.

QUOTE:

Workaround

    Right-click on AltDrag.exe > Properties > Compatibility tab
    Check the box for "Disable display scaling on high DPI displays"
    Restart Alt-Drag

Now Alt-Drag behaves correctly no matter where in the window I start dragging from, even at native resolution.

**Comment by reikred** [comment]
Hmm, I'm going to re-open this, because after reboot, the errant behavior is back. Might be because of how run-at-boot setup works??

**Comment by stefansundin** [comment]
This is almost certainly because of the DPI settings. Did the workaround stop working after a reboot?

**Comment by reikred** [comment]
Yes, I just did another test reboot. The errant behavior is back. If I then exit AltDrag (using tray and right-click menu), then start it again, it works.

So it basically seems that the start-at-boot somehow does not get or adhere to the Compatibility tab setting.

**Comment by stefansundin** [comment]
Maybe you are elevating on startup and that somehow does not follow the compatibility settings?

**Comment by reikred** [comment]
I do not have the "elevate" config setting enabled, that is all I know. Is there reason to believe that processes started by root (Admin) ignore the Compatibility settings of an executable file?

**Comment by stefansundin** [comment]
I will do more testing on this, but right now I don't have a solution. If anyone else has any input, let us know!

**Comment by daveola** [comment]
This happens on my setup but only with some windows.  I am on Windows 10.  It happens regularly with Putty.exe windows, which also have the errant behavior of not being chosen for a move if I click on an area where Putty is on top of another window - the lower window ends up being moved instead.  I tried the compatibility scaling fix and it did not help.

**Comment by pknowles** [comment]
I'm seeing this as well. Windows 10. Often with steam windows, which are scaled by 2x due to high dpi settings (my own fault really. have a 4k screen and had to get win 10 purely for application scaling).

**Comment by chrisdtaylor** [comment]
I don't see the Compatibility tab in the settings.  I have win10 and AltDrag 1.1.  Did it go away?  How else can one apply the suggested workaround?

Workaround

Right-click on AltDrag.exe > Properties > Compatibility tab
Check the box for "Disable display scaling on high DPI displays"
Restart Alt-Drag

**Comment by gcurchod** [comment]
The settings I had to change were slightly different for me (Windows 10 Pro version 1803):
Right-click on AltDrag.exe > Properties > Compatibility tab > Chang high DPI settings:
Check the box for "Overrride high DPI scaling behavior. Scaling performed by: Application".

**Comment by diradev01** [comment]
> The settings I had to change were slightly different for me (Windows 10 Pro version 1803):
> Right-click on AltDrag.exe > Properties > Compatibility tab > Change high DPI settings:
> Check the box for "Override high DPI scaling behavior. Scaling performed by: Application".

it works to me. thanks !

**Comment by basocjk** [comment]
Display Scale at 125% also causes this problem.
the said workaround also works.
